src/components/pane/pane.jsx

```diff
@@ -23,28 +23,28 @@ class Pane extends PureComponent {
   renderTabs() {
     return (
       <ul className="pane__tabs">
-        {this.props.tabs.map((tab, index) => (
+        { this.props.tabs.map((tab, index) => (
           <li
             key={ index }
-            className={ classNames('pane__tab', { 'pane__tab--is-active': this.state.activeTab === index })}
-            style={{ width: `${100 / this.props.tabs.length}%` }}
+            className={ classNames('pane__tab', { 'pane__tab--is-active': this.state.activeTab === index }) }
+            style={ { width: `${100 / this.props.tabs.length}%` } }
```

Comment:
I think this should be moved outside of `map` function since it can be calculated once instead of doing same calculation on every iteration.